client/app/connect/connect.controller.js

```diff
@@ -1,36 +1,55 @@
 angular.module('OpenEXP')
+
+    // passing services here as an array allows the the controller to be minified without losing track of the services
     .controller('ConnectCtrl', ['$scope','boardFactory', 'simulatorFactory', ($scope, boardFactory, simulatorFactory) => {
-        var _ = require('lodash');
 
-        $scope.connect = simulatorFactory.connect;
+        // grab scope for debugging
+        window.MY_SCOPE = $scope;
```

Comment:
i was just using it to quickly check the scope for variables and functions that I wanted to check were there.  im all ears if you've got a different approach